src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -76,14 +76,23 @@ export default class EthereumBlockchainService implements BlockchainService {
     } else {
       const gasPriceEstimate = await this.provider.getGasPrice();
       // Add 10% extra to gas price for each subsequent attempt
-      txData.gasPrice = gasPriceEstimate.add(gasPriceEstimate.mul(attempt * .01));
+      txData.gasPrice = EthereumBlockchainService.increaseGasPrice(gasPriceEstimate, attempt)
       logger.debug('Estimated Gas price: ' + txData.gasPrice.toString());
 
       txData.gasLimit = await this.provider.estimateGas(txData);
       logger.debug('Estimated Gas limit: ' + txData.gasLimit.toString());
     }
+  }
 
-
+  /**
+   * Takes current gas price and attempt number and returns new gas price with 10% increase per attempt
+   * @param currentGas
+   * @param attempt
+   */
+  static increaseGasPrice(currentGas: BigNumber, attempt: number): BigNumber {
```

Comment:
Can we change method name to something like `updateGasPriceByAttempt` ?

Reply:
Renamed to `increaseGasPricePerAttempt`